packages/snap/src/utils/network.ts

```diff
@@ -0,0 +1,47 @@
+import { config } from '@hathor/wallet-lib';
+import { REQUEST_METHODS, DEFAULT_NETWORK, NETWORK_MAP } from '../constants';
+
+/*
+ * Get persisted network data or the default from constants
+ */
+export const getNetworkData = async () => {
+  const persistedData = await snap.request({
+    method: REQUEST_METHODS.MANAGE_STATE,
+    params: { operation: 'get' },
+  }) ?? {};
+
+  const network = persistedData.network ?? DEFAULT_NETWORK;
```

Comment:
question: Maybe this is out of the scope of this feature, but: Is this the full network name?

For example, if the request tries to change between `testnet` and `nano-testnet` will this show any difference between the two?

Reply:
Yes, it's different because I have different network URLs for each of these networks. It depends on what I want to support in the constants file.

```
export const NETWORK_MAP = {
  'mainnet': MAINNET_URLS,
  'nano-bravo-testnet': NANO_BRAVO_URLS,
  'testnet': TESTNET_URLS,
  'testnet-hotel': TESTNET_URLS,
}
```